pkg/ccl/streamingccl/streamingest/stream_ingestion_job.go

```diff
@@ -68,22 +68,19 @@ func getStreamIngestionStats(
 		lagInfo := &streampb.StreamIngestionStats_ReplicationLagInfo{
 			MinIngestedTimestamp: *highwater,
 		}
-		lagInfo.SlowestSourcePartitionTimestamp = hlc.MaxTimestamp
```

Comment:
How much do we care about per-partition progress in this way?  Should I bring them back?